dll/cpl/mmsys/sounds.c

```diff
@@ -1182,7 +1209,13 @@ SoundsDlgProc(HWND hwndDlg,
 
                     if (GetOpenFileNameW(&ofn))
                     {
-                        // FIXME search if list already contains that sound
+                        // search if list already contains that sound
+                        lResult = FindSoundFileInList(hwndDlg, filename);
+                        if (lResult != CB_ERR)
```

Comment:
```suggestion
                        if (lResult >= 0)
```
Seems a bit more intuitive, but fine either way.

Reply:
I think `CB_ERR` is more readable.